docs/tutorials/integrating-with-laravel.md

```diff
@@ -0,0 +1,85 @@
+# Integrating with Laravel
+
+This page explains how to integrate Shoelace with a [Laravel](https://laravel.com) app using a local Webpack bundle. This is a community-maintained document. For questions about this integration, please [ask the community](/resources/community).
+
+## Requirements
+
+This integration has been tested with the following:
+
+- Laravel >= 8
+- Node >= 14
+- Laravel Mix >= 6
+
+## Instructions
+
+These instructions assume an out-of-the-box [Laravel 8+ install](https://laravel.com/docs/8.x/installation), using [Laravel Mix](https://laravel.com/docs/8.x/mix) to compile assets.
+Be sure to run `npm install` to install the default Laravel front-end dependencies before installing Shoelace. 
+
+### Install the Shoelace package
+```bash
+npm install @shoelace-style/shoelace
+```
+
+### Import the Default Theme
+
+Import Shoelace's default theme (stylesheet) in `/resources/css/app.css`:
+
+```css
+@import "/node_modules/@shoelace-style/shoelace/dist/themes/light.css";
+```
+
+### Import Required Scripts
+
+Import each Shoelace component you plan to use in `/resources/js/boostrap.js`:
+
+```js
+import { SlAlert, SlAnimation, SlButton, ... } from '@shoelace-style/shoelace'
```

Comment:
Components are registered via side effects and I'm pretty sure webpack will load the entire library even if you only import a single component this way (i.e. using the bare module specifier). I'd love to be wrong about this but, last I checked, the behavior was how I described it here.

It's unfortunate, but we may need to [cherry pick using the full path](https://shoelace.style/getting-started/installation?id=cherry-picking) for each component to prevent a bloated webpack bundle.

Reply:
Looks like Webpack still behaves the same way. Using the full path for each component cut the bundle size of the `vendor.js` in half -- at least in my example app.

Reply:
It's probably worth updating the imports then — and maybe calling out that a copy/paste link for components is available at the bottom of each docs page.

Reply:
Done. Let me know if you see the updates.